x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/dependencies.spec.ts

```diff
@@ -43,6 +44,17 @@ describe('Dependencies', () => {
 
       cy.contains('h1', 'postgresql');
     });
+
+    it('has no detectable a11y violations on load', () => {
+      cy.visit(
+        `/app/apm/services/opbeans-java/dependencies?${new URLSearchParams(
+          timeRange
+        )}`
+      );
+      cy.contains('a[role="tab"]', 'Dependencies');
```

Comment:
If there is a second tab that can be interacted with, it'd be great to click on that tab and make a second `checkA11y` call to bump code coverage.

Reply:
Hi @1Copenut, this is our first approach to the A11y checks in our pages, we are going to add more checks in the future for now we are just checking the first render and not the interactions.

APM Service Overview contains multiple tabs, it's good to know that we can bump the code coverage if we check A11y  in all of them.